### DISCUSSION TITLE
docs: *less -> README.md + /docs/index.md

### DISCUSSION THREAD

**Discussion by vobu** [ui5-community/ui5-ecosystem-showcase#520] [General]
hi @marcelschork + @sebbi08 - just a hint (which is not "worth" an issue, thus a discussion item): might want to include your `less` middleware + task in the respective .md files: README.md + /docs/index.md

**Comment by marcelschork** [comment]
hi @vobu,
once it's published on npm I will add it to the respective files.

FYI @sebbi08

**Reply by marcelschork** [reply]
#522 opened
